enterprise/backend/test/metabase_enterprise/enhancements/integrations/ldap_test.clj

```diff
@@ -213,7 +213,7 @@
             (db/delete! User :%lower.email "john.smith@metabase.com")))))))
 
 (deftest fetch-or-create-user-test
-  (with-redefs [premium-features/enable-enhancements? (constantly true)]
+  (with-redefs [#_{:clj-kondo/ignore [:deprecated-var]} premium-features/enable-enhancements? (constantly true)]
```

Comment:
you don't have to change it but does this ignore directive work on the line above or must it be inline like this?

Reply:
it does work on the line above, but from my understanding it will then ignore everything in the entire form. So if I do it right here it's literally ignoring just one single form rather than potentially accidentally ignoring some additional stuff on top of that